src/gl/texture.js

```diff
@@ -86,7 +87,7 @@ export default class Texture {
     }
 
     bind(unit = 0) {
-        if (!this.valid) {
+        if (!this.valid || debugSettings.wireframe) {
```

Comment:
Can you tell me what this check is about? Shouldn't any textures used by the geometry still be valid, even if they are not being used for rendering?

Reply:
I'm getting hundreds of these WebGL errors in Chrome with this:

```
WebGL: INVALID_OPERATION: texImage2D: no texture bound to target
WebGL: INVALID_OPERATION: texParameter: no texture bound to target
[.WebGL-0x7f8b71f16e00]RENDER WARNING: there is no texture bound to the unit 0
```

I'm not getting any errors with this check removed though, e.g. reverted to original code.

Reply:
OK I think I see what you were trying to achieve with this: to get textured points to render as quads, instead of wireframes masked against a sprite?

I will try to think of another way to accomplish this, but we can't disable texture binding this low-level.

Reply:
Yep that's the goal – though tbh I've never needed to inspect sprite geometry, I just wanted to try this for completeness

Reply:
OK this is my attempt to make these more visible, think it works alright: https://github.com/tangrams/tangram/pull/745/commits/da98a78c01051600ed17c7df8c8ab08a4344a088